### PR TITLE
Mark all input errors from LPMS as non-retriable

### LINKED ISSUE #1756: Mark additional LPMS transcode errors as non-retryable on B
The LPMS [error map](https://github.com/livepeer/lpms/blob/master/ffmpeg/ffmpeg_errors.go#L36) contains a list of errors that can indicate that transcoding of a segment is not possible. There is a good chance that if an O returns one of these errors that retrying the segment with another O will not help because the transcoder may not support a particular type of segment. B already considers one of these errors ("No keyframes in input") as non-retryable. B might be able to consider all of these errors as non-retryable and avoid needlessly trying segments that trigger these errors with other Os.

### PR DESCRIPTION
**Pull request by jailuthra**
**What does this pull request do? Explain your changes. (required)**

Mark all input errors from LPMS as non-retriable

**Specific updates (required)**

- Update LPMS module
- Go through all LPMS errors and mark segment as non retriable if it returns an error in the [known list](https://github.com/livepeer/lpms/pull/224/files#diff-1e183f602cccda6ecfda8743a53256a6ed89486c0b3b4ce123b6a23295fcdfe1R17-R21).

**How did you test each of these updates (required)**

Manually tested with a segment with invalid pixel format https://cdn.discordapp.com/attachments/426409080551374849/809890640527097862/1.ts - got the correct behaviour -
```
I0216 20:10:17.532951  368785 segment_rpc.go:465] Uploaded segment nonce=6549207908557319073 manifestID=test sessionID=ab710a7e seqNo=2 orch=https://0.0.0.0:8935 dur=2.848236ms
E0216 20:10:17.539248  368785 segment_rpc.go:497] Transcode failed for segment nonce=6549207908557319073 manifestID=test sessionID=ab710a7e seqNo=2 orch=https://0.0.0.0:8935 err=Unsupported input pixel format
W0216 20:10:17.539286  368785 broadcast.go:440] Not retrying current segment nonce=6549207908557319073 seqNo=2 due to non-retryable error err=Unsupported input pixel format
E0216 20:10:17.539331  368785 mediaserver.go:877] http push error processing segment url=http://127.0.0.1:8936/live/test/2.ts manifestID=test err=Unsupported input pixel format
```

**Does this pull request close any open issues?**

Fixes https://github.com/livepeer/go-livepeer/issues/1756

**Checklist:**

- [ ] README and other documentation updated
- [ ] Node runs in OSX and devenv
- [x] All tests in `./test.sh` pass

### CODE REVIEW

**Review by darkdarkdragon — APPROVED** [review]
LGTM

**Review by yondonfu — APPROVED** [review]
LGTM after merging https://github.com/livepeer/lpms/pull/224 Not sure if go mod needs to be updated in case the commit hash is different on master in LPMS?

**Comment by jailuthra** [comment]
> LGTM after merging [livepeer/lpms#224](https://github.com/livepeer/lpms/pull/224) Not sure if go mod needs to be updated in case the commit hash is different on master in LPMS?

Thx! Did a manual rebase+push in LPMS to retain the same commit ID in master.

**Pull request merged**
